api/routes/assessment_routes.py

```diff
@@ -114,3 +115,103 @@ def get_sub_criteria_theme_answers(application_id: str, theme_id: str):
+    round_id: str
+) -> List[Dict]:
+    """
+    Function used by the endpoint
+    `/assessments/get-stats/{fund_id}/{round_id}`
+    that returns a dictionary of metrics about
+    assessments for a given fund_id and round_id.
+
+    :param fund_id: The stringified fund UUID.
+    :param round_id: The stringified round UUID.
+    :return: A list of dictionaries.
+    """
+    stats = {}
+    assessments = get_metadata_for_fund_round_id(
+        fund_id=fund_id,
+        round_id=round_id
+    )
+    qa_completed_assessments = [
+        flag["application_id"] for flag in get_latest_flags_for_each(
+        "QA_COMPLETED"
+    )
+    ]
+    stopped_assessments = [
+        flag["application_id"] for flag in get_latest_flags_for_each(
+            "STOPPED"
+        )
+    ]
+    flagged_assessments = [
+        flag["application_id"] for flag in get_latest_flags_for_each(
+            "FLAGGED"
+        )
+    ]
+    # TODO: add resolved if required, after flag_type enum
+    # has been updated to include QUERY_RESOLVED
+    # resolved_assessments = [
+    #     flag["application_id"] for flag in get_latest_flags_for_each(
+    #         "QUERY_RESOLVED"
+    #     )
+    # ]
+    stats.update(
+        {
+            "completed": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["workflow_status"] == "COMPLETED"
+                ]
+            ),
+            "assessing": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["workflow_status"] == "ASSESSING"
+                ]
+            ),
+            "not_started": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["workflow_status"] == "NOT_STARTED"
+                ]
+            ),
+            "qa_completed": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["application_id"] in qa_completed_assessments
+                ]
+            ),
+            "stopped": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["application_id"] in stopped_assessments
+                ]
+            ),
+            "flagged": len(
+                [
+                    1
+                    for assessment in assessments
+                    if assessment["application_id"] in flagged_assessments
+                ]
+            ),
+    # TODO: add resolved if required, after flag_type enum
+    # has been updated to include QUERY_RESOLVED
+    #         "query_resolved": len(
+    #             [
+    #                 1
+    #                 for assessment in assessments
+    #                 if assessment["application_id"] in resolved_assessments
+    #             ]
+    #         ),
```

Comment:
Have just left this commented for now pending update of FlagType enum to contain QUERY_RESOLVED fyi @Adam-W1

Reply:
Removed